Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -65,5 +71,36 @@ public class InAppMessageView: UIView {
         //      }
     }
 
-    private func displayInAppMessage(_ message: Message) {}
+    private func displayInAppMessage(_ message: Message) {
+        // There might already be a message displayed. If so, remove it and cleanup the resources.
+        // First, remove the subview that the inline manager has a reference to.
+        // Lastly, cleanup the inline manager.
+        subviews.first?.removeFromSuperview()
+        inlineMessageManager = nil
```

Comment:
// TODO as we talked about in the call today, I'll comment this out and we can implement this in a future PR. 

I was trying to be resourceful by cleaning up old resources but I can see how this would change the functionality of the View. We decided to implement this later.

Reply:
Done. Added a `guard` statement to exit early if we are already showing a message.